core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -398,6 +399,7 @@
         "alluxio.underfs.oss.connection.timeout.ms";
     public static final String UNDERFS_OSS_CONNECT_TTL = "alluxio.underfs.oss.connection.ttl";
     public static final String UNDERFS_OSS_SOCKET_TIMEOUT = "alluxio.underfs.oss.socket.timeout.ms";
+    public static final String UNDERFS_S3A_ENABLE_ACL = "alluxio.underfs.s3a.enable.acl";
```

Comment:
Could you document this new configuration option? Also, this may be better named like `inherit_acl`.